src/pool/disjoint/pool_disjoint_impl.cpp

```diff
@@ -815,16 +825,36 @@ void *DisjointPool::AllocImpl::allocate(size_t Size, size_t Alignment,
     return nullptr;
 }
 
-Bucket &DisjointPool::AllocImpl::findBucket(size_t Size) {
+std::size_t DisjointPool::AllocImpl::sizeToIdx(size_t Size) {
```

Comment:
is there any speed-up after doing this?

Reply:
Compared to the linear search there is a noticeable speed up. From what I've seen in microbenchmark it's faster in all cases , the larger the allocation size, the bigger the difference, but even for allocation sizes like 64 or 128 (when MinBucketSize equals 8), it's approx. 4 times faster.